### PR TITLE
corrected zh-hans.json of Volo.Abp.UI localization

### PR DESCRIPTION
**Pull request by MrChuJiu**

### CODE REVIEW

File: framework/src/Volo.Abp.UI/Localization/Resources/AbpUi/zh-Hans.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "Clear": "清除",
     "Yes": "是",
     "No": "否",
-    "Ok": "Ok",
+    "Ok": "好",
```
Comment (realLiangshiwei) [inline framework/src/Volo.Abp.UI/Localization/Resources/AbpUi/zh-Hans.json:10]: `确定` more better than `好`
Like: https://ant.design/components/modal/

Comment (wangj90) [reply framework/src/Volo.Abp.UI/Localization/Resources/AbpUi/zh-Hans.json:10]: If refer to ant design, it seems to be 知道了。
image

Comment (MrChuJiu) [reply framework/src/Volo.Abp.UI/Localization/Resources/AbpUi/zh-Hans.json:10]: '好', '知道了', '确定' all match the translation

Comment (realLiangshiwei) [reply framework/src/Volo.Abp.UI/Localization/Resources/AbpUi/zh-Hans.json:10]: it doesn't matter

**Review by realLiangshiwei — APPROVED** [review]

**Comment by realLiangshiwei** [comment]
Thanks @MrChuJiu

**Pull request merged**